scripts/lib/wic/canned-wks/mkefidisk-multiboot.wks

```diff
@@ -10,4 +10,6 @@ part / --source rootfs --ondisk sda --fstype=ext4 --label platform --align 1024
 
 part swap --ondisk sda --size 44 --label swap1 --fstype=swap
 
+part /media --ondisk sda --size 787 --fstype=ext4 --label media --align 1024
```

Comment:
Why 787? why not 1 GB? If you are doing this due to overhead factor, then it will be good to explicitly use the overhead option so anyone looking at this file can understand the reasoning behind this particular size. I think it will be better to use 1 GB directly and set the factor to 1.

Reply:
When i set 787 i expected the multiplier by 1.3 (so that it reaches 1 GB) but that didnt happen for this partition. Currently the image size is 7.9 GB and if i set this partition  to 1 GB, then it will be ~8.2 GB. Since we changed the requirements to a 16GB USB, then i'll change this.